comp/core/workloadmeta/def/types.go

```diff
@@ -584,6 +587,7 @@ func (c Container) String(verbose bool) string {
 		_, _ = fmt.Fprintln(&sb, "Network IPs:", mapToString(c.NetworkIPs))
 		_, _ = fmt.Fprintln(&sb, "PID:", c.PID)
 		_, _ = fmt.Fprintln(&sb, "Cgroup path:", c.CgroupPath)
+		_, _ = fmt.Fprintln(&sb, "GPU Type:", c.Resources.GPUType)
```

Comment:
There's this some lines above: 
```go
	_, _ = fmt.Fprintln(&sb, "----------- Resources -----------")
	_, _ = fmt.Fprint(&sb, c.Resources.String(verbose))
```

So I think this one is redundant, but we'll need to update `func (cr ContainerResources) String(bool) string` to also print the GPUType.